pkg/registrar/map_functions.go

```diff
@@ -286,3 +288,424 @@ func runTemplateRefMatch(ref v1alpha1.TemplateReference, runTemplate *v1alpha1.C
 
 	return ref.Kind == "ClusterRunTemplate" || ref.Kind == ""
 }
+
+func (mapper *Mapper) ServiceAccountToWorkloadRequests(serviceAccountObject client.Object) []reconcile.Request {
```

Comment:
this file might be due for a refactor due to size - future

Reply:
Yeah big +1 for doing this soon after landing the feature. Could refactor funcs to get more re-use on the Role/ClusterRole/RoleBinding/ClusterRoleBinding handling funcs. Does make me wonder though, if it makes more sense to have the registrar set up a group of 'global' informers for the RBAC objects?